src/Cache/CacheManager.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Winter\Storm\Cache;
+
+use Illuminate\Cache\CacheManager as BaseCacheManager;
+
+class CacheManager extends BaseCacheManager
```

Comment:
@bennothommo is this still required?

Reply:
@LukeTowers I checked in Storm and Winter source code and I couldn't see a reference to this class... I think it can be safely removed.